docs/architecture/solana/migration-faq.mdx

```diff
@@ -8,16 +8,14 @@ image: https://docs.helium.com/img/link-image.png
 slug: /solana/migration-faq
 ---
 
-import LegacyContentBanner from '@site/src/theme/LegacyContentBanner'
-
-<LegacyContentBanner />
-
 # FAQ - Solana Migration
 
 ## Why Does My Account Have A Small Amount Of SOL?
 
 Every migrated Wallet was seeded with a small amount, approximately 0.00139, of SOL for
-transactions, enough for roughly 100 transactions.
+transactions, enough for roughly 100 transactions. The amount of SOL needs to be topped up and we
+recommmend 0.1 SOL and keeping it above 0.02 or Hotspot, Token exchange, staking and voting 
+transactions attempted on the blockchain may fail. 
```

Comment:
It would be good to refactor this section to read a bit more clearly. 
Too many commas, and run-on sentences, make the reading experience a bit challenging, for the user.